cmd/cert-checker/main.go

```diff
@@ -122,6 +131,9 @@ func (c *certChecker) getCerts(unexpiredOnly bool) error {
 		if err != nil {
 			return err
 		}
+		if len(certs) == 0 {
```

Comment:
Can you move this check to right before the potential bounds breakage (i.e. `certs[len(certs)-1]`)?